src/Polly.Strategies/Timeout/TimeoutTaskArguments.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Polly.Timeout;
+
+public readonly struct TimeoutTaskArguments
```

Comment:
Why is Task in the name?

Reply:
Maybe TimeoutStrategyArguments? Then you could use the same pattern for RetryStrategyArguments, etc.

Reply:
I think the arguments class should follow the name of the event where it is used. The property is called `OnTimeout` so my vote would be `OnTimeoutArguments`. 

The task stuff is just a copy-pate error :)